### ISSUE TITLE
Scale down pods which are running twice

### ISSUE THREAD

**Issue by cfergeau** [code-ready/snc#178]
Gorkem spotted a few pods which are running twice and
which should not be needed. I'm filing an issue so that we don't forget about
it

**Comment by cfergeau** [comment]
Proof of concept patch, but not enough as indicated by the commit log ;)

```
commit 275a319defd0d67084af0f43a79ccbe64a33b065
Author: Christophe Fergeau <cfergeau@redhat.com>
Date:   Fri May 15 09:47:46 2020 +0200

    Scale a few pods down to 1
    
    Gorkem spotted a few pods which are running twice. This commit tries to
    scale them down to one. 2 of them are scaled back to 2, but I did not
    investigate what was doing it/if there's a way around it.

diff --git a/snc.sh b/snc.sh
index 3f2f42e..2c3acd6 100755
--- a/snc.sh
+++ b/snc.sh
@@ -311,5 +311,12 @@ ${OC} delete apiservice v1beta1.metrics.k8s.io
 # Scale route deployment from 2 to 1
 ${OC} patch --patch='{"spec": {"replicas": 1}}' --type=merge ingresscontroller/default -n openshift-ingress-operator
 
+${OC} scale --replicas=1 -n openshift-console deployments/console
+${OC} scale --replicas=1 -n openshift-operator-lifecycle-manager deployments/packageserver
+
+# These 2 are scaled back to 2 :(
+${OC} scale --replicas=1 -n openshift-console deployments/downloads
+${OC} scale --replicas=1 -n openshift-authentication deployments/oauth-openshift
+
 # Set default route for registry CRD from false to true.
 ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge
```

**Comment by gbraad** [comment]
Likely the CVO will reconcile by scaling them back to 2

**Comment by praveenkumar** [comment]
This will be fixed automatic for 4.8 since it will only run single replica for each resource.
